x/bank/app_test.go

```diff
@@ -365,21 +404,22 @@ func TestMsgMultiSendDependent(t *testing.T) {
 }
 
 func TestMsgSetSendEnabled(t *testing.T) {
+	t.Skip("skipping test as not relevant to Heimdall (MsgSetSendEnabled is not required as the only denom supported is matic)")
```

Comment:
We haven't enabled `MsgSetSendEnabled` , so that serves as a "check" (for instance we aren't [registering the message in the codec](https://github.com/0xPolygon/cosmos-sdk/pull/5/files#diff-265e2c1f4bb00275a0d7c51902fadc3f121a5b2eefa5b3d96f85e755a9d34aa4L18) ). 
Also IIRC @glnro that this will be deprecated going forward.